runtime/rlog/rlog.go

```diff
@@ -27,6 +26,9 @@ const (
 	levelInfo  logLevel = 2
 	levelWarn  logLevel = 3
 	levelError logLevel = 4
+
+	//publicapigen:drop
+	InternalKeyPrefix = "encore_"
```

Comment:
Move this to a separate `const` declaration; this one is used for log levels and it makes more sense to keep the  `InternalKeyPrefix ` constant separate. Also, it would be nice to have a doc comment for it.